packages/upload-client/src/unixfs.js

```diff
@@ -34,8 +34,14 @@ export function createFileEncoderStream(blob) {
   const unixfsWriter = UnixFS.createWriter({ writable, settings })
   const fileBuilder = new UnixFSFileBuilder('', blob)
   void (async () => {
-    await fileBuilder.finalize(unixfsWriter)
-    await unixfsWriter.close()
+    try {
+      await fileBuilder.finalize(unixfsWriter)
+      /* c8 ignore next 2 */
+    } catch (e) {
+      console.log('Error finalizing file builder: ', e)
```

Comment:
We shouldn't smother. Use `abort`? - https://github.com/ipld/js-unixfs/blob/f6cd633b3dec7ef24483ce7c4d4464cbd20bc78b/src/writer/api.ts#L30

Reply:
Ah that's the magic! done, plus improved the testing situation - struggled to find a way to test the remaining `c8 ignore next` bit but came up short - it may actually be rare/impossible to trigger an error in that flow so I feel ok about this.